src/main/java/core/basesyntax/service/impl/ReaderServiceImpl.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.ReaderService;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+
+public class ReaderServiceImpl implements ReaderService {
+    private static final int INDEX_TITLE_LINE = 0;
+
+    @Override
+    public List<String> readFromFile(String fileName) {
+        List<String> lines;
+        try {
+            lines = Files.readAllLines(Paths.get(fileName));
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read from file " + fileName, e);
+        }
+        lines.remove(INDEX_TITLE_LINE);
```

Comment:
let's remove the first line in parser or in Main
we should be able to read from any file with this method